hack/get-default-sa-token.sh

```diff
@@ -1,5 +1,3 @@
 #!/bin/bash
 
-SECRET_NAME=$(kubectl get serviceaccount default -n default -o jsonpath='{.secrets[0].name}')
-
-kubectl get secret "${SECRET_NAME}" -o jsonpath='{.data.token}' | base64 -d
+kubectl create token default -n default
```

Comment:
I know I'm not using the latest greatest but I'd like us to be a little bit more robust here, if possible...

```
$ kubectl version                                                                             
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.3", GitCommit:"2e7996e3e2712684bc73f0dec0200d64eec7fe40", GitTreeState:"clean", BuildDate:"2020-05-20T12:52:00Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"23", GitVersion:"v1.23.1", GitCommit:"86ec240af8cbd1b60bcc4c03c20da9b98005b92e", GitTreeState:"clean", BuildDate:"2021-12-16T11:34:54Z", GoVersion:"go1.17.5", Compiler:"gc", Platform:"linux/amd64"}

$ kubectl create token default -n default                                             
Error: must specify one of -f and -k

error: unknown command "token default"
See 'kubectl create -h' for help and examples
```

Reply:
updated. I think I've made it simpler